appng-core/src/test/java/org/appng/core/controller/RepositoryWatcherTest.java

```diff
@@ -66,7 +67,13 @@ public void test() throws Exception {
 
 		FileUtils.touch(new File(rootDir, fehlerJsp));
 		FileUtils.touch(new File(rootDir, testJsp));
-		Thread.sleep(200);
+		if (OperatingSystem.isMac()) {
```

Comment:
Better: use `@Test(timeout = 20000)` on the `test()` method, then do this:
```
		while (ehcache.getSize() != 0) {
			Thread.sleep(100);
		}
```

In that way the test remains OS-agnoistic.

Reply:
Thank you for your hints, I amended the pull request and tested on Mac OS X.